packages/components/src/menu/stories/index.story.tsx

```diff
@@ -64,300 +64,337 @@ const meta: Meta< typeof Menu > = {
 			source: { excludeDecorators: true },
 		},
 	},
-};
+} satisfies Meta< typeof Menu >;
 export default meta;
 
-export const Default: StoryFn< typeof Menu > = ( props: MenuProps ) => (
-	<Menu { ...props }>
-		<Menu.TriggerButton
-			render={ <Button __next40pxDefaultSize variant="secondary" /> }
-		>
-			Open menu
-		</Menu.TriggerButton>
-		<Menu.Popover>
-			<Menu.Item>
-				<Menu.ItemLabel>Label</Menu.ItemLabel>
-			</Menu.Item>
-			<Menu.Item>
-				<Menu.ItemLabel>Label</Menu.ItemLabel>
-				<Menu.ItemHelpText>Help text</Menu.ItemHelpText>
-			</Menu.Item>
-			<Menu.Item>
-				<Menu.ItemLabel>Label</Menu.ItemLabel>
-				<Menu.ItemHelpText>
-					The menu item help text is automatically truncated when
-					there are more than two lines of text
-				</Menu.ItemHelpText>
-			</Menu.Item>
-			<Menu.Item hideOnClick={ false }>
-				<Menu.ItemLabel>Label</Menu.ItemLabel>
-				<Menu.ItemHelpText>
-					This item doesn&apos;t close the menu on click
-				</Menu.ItemHelpText>
-			</Menu.Item>
-			<Menu.Item disabled>Disabled item</Menu.Item>
-			<Menu.Separator />
-			<Menu.Group>
-				<Menu.GroupLabel>Group label</Menu.GroupLabel>
-				<Menu.Item prefix={ <Icon icon={ customLink } size={ 24 } /> }>
-					<Menu.ItemLabel>With prefix</Menu.ItemLabel>
-				</Menu.Item>
-				<Menu.Item suffix="⌘S">With suffix</Menu.Item>
-				<Menu.Item
-					disabled
-					prefix={ <Icon icon={ formatCapitalize } size={ 24 } /> }
-					suffix="⌥⌘T"
-				>
-					<Menu.ItemLabel>
-						Disabled with prefix and suffix
-					</Menu.ItemLabel>
-					<Menu.ItemHelpText>And help text</Menu.ItemHelpText>
-				</Menu.Item>
-			</Menu.Group>
-		</Menu.Popover>
-	</Menu>
-);
-Default.args = {};
+type Story = StoryObj< MenuProps >;
```

Comment:
Any reason it's not this?

```suggestion
type Story = StoryObj< typeof Menu >;
```

Reply:
Same as https://github.com/WordPress/gutenberg/pull/68204#discussion_r1894949993 — performance reasons.

Although I'm happy to revert if we don't think performance is improved with this change.